src/ConsoleMain/Main.cpp

```diff
@@ -68,7 +70,10 @@ int main(int argc, char** argv)
         ("out_file_inclusion_graph", "Path to output file inclusion graph", cxxopts::value(outputPathFileInclusionGraph))
         ("out_file_compilations", "Path to output file compilations data", cxxopts::value(outputPathFileCompilations))
         ("out_build_timeline", "Path to output build timeline", cxxopts::value(outputPathBuildTimeline))
-        ("out_template_instantiations", "Path to output template instantiations data", cxxopts::value(outputPathTemplateInstantiations));
+        ("out_template_instantiations", "Path to output template instantiations data", cxxopts::value(outputPathTemplateInstantiations))
+        //timing display
+        ("timing", "defaults to ns, can use ms(miliseconds) or s(seconds)", cxxopts::value(timingDisplay))
```

Comment:
Although I like `timing`, it doesn't feel related to the time unit. How about giving it a name that makes it crystal clear?

Also, some spell-check and tweaks:
- Description should start in a capital letter.
- `miliseconds` -> `milliseconds`.
- Consider using `ns(nanoseconds)`, as you've used that format with the other units.

Finally, as mentioned in previous comment, if you consider making `s(seconds)` the default, please update the description.

Reply:
Naming is hard :(
display_time_unit ?
the rest are fixed.

Reply:
Yeah, `display_time_unit` works!
Please go with it :)